streams/src/test/java/org/apache/kafka/streams/TopologyTest.java

```diff
@@ -157,7 +156,7 @@ public void testNamedTopicMatchesAlreadyProvidedPattern() {
     }
 
     @Test
-    public void shouldNotAllowToAddProcessorWithSameName() {
+    public void shoudNotAllowToAddProcessorWithSameName() {
```

Comment:
Why this?

Reply:
I think this is a typo that can be reverted.